<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="plotwidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>751</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearplot">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить график</string>
    </property>
   </widget>
   <widget class="QLabel" name="position">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="filtration" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>861</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="knopka"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="characteristics_chose">
    <property name="title">
     <string>Характеристика</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Фильтрация</string>
    </property>
    <addaction name="action_X"/>
    <addaction name="action_Y"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="characteristics_chose"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="knopka">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>Выбор диапозона по оси X</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="text">
    <string>Выбор диапозона по оси Y</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
